### ISSUE TITLE
Feature: Do not move armor over to show durability of weapons

### ISSUE THREAD

**Issue by RacerDelux** [61352151511/Durability-Show#3]
Currently the hud shifts to the side when you hover over a weapon. This takes up a lot of the screen space! Can you make it so it simply shifts up? (or add the option to switch between the two)

**Comment by ElvenChaos** [comment]
Another alternative I'd like available: if you have your hud on the right side and you hover over a weapon or item with durability, could that particular item pop up towards the center instead of the item pushing the armor towards the center? Currently, the armor being misplaced is jarring.

**Comment by Zygus42** [comment]
I agree with this suggestion - was about to add it myself.  (using 1.12).  Keep the armor Hud in the same place, even when equipping a weapon.
